app/controllers/workflows_controller.rb

```diff
@@ -48,7 +48,8 @@ def create
       processes: initial_parser.processes,
       version: Version.new(
         druid: params[:druid],
-        version: params[:version]
+        version: params[:version],
+        metadata: params[:metadata]
```

Comment:
I pushed a commit which parses the incoming JSON in the controller as you suggested instead of in the `Version` class (which now accepts a `hash` instead of a JSON string).  I wrote a new spec to show that if you pass in something that is not parseable to the API, it will blow up.    Is that what you were thinking?

Reply:
spec added: https://github.com/sul-dlss/workflow-server-rails/pull/745/files#diff-8dbb6de325a7f83cbcaa0ba40bcd48e3fa545771136f9cd8a50d540eee018364R63-R75